nengo-data/nengorc

```diff
@@ -72,6 +78,11 @@
 # --- Settings for the Nengo core simulator
 [nengo.Simulator]
 
+# NumPy dtype that will be used for all Simulator signals, if possible.
+# Note that Nengo is only expected to work for float dtypes.
+# If set to 'auto', will use the value in 'precision.dtype'. (string)
+#dtype: auto
```

Comment:
Yeah I would also be cool with removing it; I kept it in because @hunse had it in his commit message, and I believe @arvoelke asked about it at some point, so I'd like their thoughts on whether it's necessary.

Reply:
For me, it's important to be able to make two simulators in the same script with different dtypes. However, I don't think that this line is necessary for that.

Aaron was asking about having a different dtype for build and simulation. This might be something we want to do in the future, but unfortunately it's never been supported by this PR.

So I think it's fine to get rid of this.